Assets/Materials/Television_var.mat

```diff
@@ -26,5 +26,5 @@ Material:
     m_Ints: []
     m_Floats: []
     m_Colors:
-    - _Color: {r: 0.3058179, g: 0.3058179, b: 0.3058179, a: 0}
+    - _Color: {r: 0.30581748, g: 0.30581748, b: 0.30581748, a: 0}
```

Comment:
unnecessary change

Reply:
Yeah, I don't actually know why that was changed. If I remember correctly, I didn't touch the asset.